react/README.md

```diff
@@ -0,0 +1,158 @@
+- [Developer Notes](#developer-notes)
+- [Available Scripts](#available-scripts)
+
+# Prerequisites 
+You will need the following things:
+
+- PingOne for Customers Account - If you don’t have an existing one, please register it.
+- An new [Application](https://documentation.pingidentity.com/pingone/p14cAdminGuide/index.shtml#p1_t_addApplication.html), configured for Single-Page App (SPA) or Native mode. Also make sure that it is enabled plus redirect, sign off URL's and access grants by scopes are properly set.
+- Specify custom login page URL for the application in __SIGNON URL__ textbox of P14C admin console. 
+
+# Getting Started
+
+1. Clone a source code
+`git clone git@github.com:pingidentity/pingone-customers-sample-custom-signon.git . `
+2. Update [config.js](./src/config.js) with your application data extracted from P14C admin console:
+    - `environment_id`: *Required*. Your application's Environment ID. You can find this value at your Application's Settings under 
+    **Configuration** tab from the admin console( extract `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` string that specifies the environment 128-bit universally unique identifier ([UUID](https://tools.ietf.org/html/rfc4122)) right from `https://auth.pingone
+    .com/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/as/authorize` 
+    *AUTHORIZATION URL* ). Or from the *Settings* main menu (*ENVIRONMENT ID* variable)
+    - `client_id`: *Required*. Your application's client UUID. You can also find this value at Application's Settings right under the 
+    Application name.
+    - `clientSecret`: A string that specifies the the application’s client secret. This property is required if your application **is not a public client**. Plus make sure to set it if the following  the application’s `tokenEndpointAuthMethod` property is set to `CLIENT_SECRET_POST`.
+    - `responseType`: A string that specifies the code or token type returned by an authorization request. Options are `token`, `id_token`, and `code`. The value **MUST** be the `code` for requesting an authorization code flow, `id_token token` for the implicit flow, or `code id_token` for hybrid flow(a scenario when you can have a long lived session in your application and get tokens back immediately from the authorization endpoint).
+    For more variants please check [Definitions of Multiple-Valued Response Type Combinations](https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html#Combinations) topic.
+    - `grantType`: A string that specifies the grant type of the token request. Options are `authorization_code`, `implicit`(is set by default), and `client_credentials`. So, if you have `responseType=code`, then define `redirectUri=authorization_code`.
+    - `redirectUri`: *Required*. The URL to which the PingOne will redirect the user's browser after authorization has been granted by the user. *REDIRECT URLS* values corresponds to this data. The Access and ID Token will be available in the hash fragment of this URL. 
```

Comment:
The URL to which PingCloud will redirect

Reply:
We are using PingOne for now